lib/firmata.js

```diff
@@ -545,6 +545,17 @@ var Board = function(port, options, callback) {
         this.setSamplingInterval(settings.samplingInterval);
       }
       if (settings.skipCapabilities) {
+        this.analogPins = this.settings.analogPins || [ 14, 15, 16, 17, 18, 19 ];
+        this.pins = this.settings.pins || this.pins;
+        if (!this.pins.length) {
+          for (var i = 0; i < (this.settings.pinCount || 20); i++) {
```

Comment:
I recommend setting the fallback to either 64 or 128. That way it will still work with a Mega or Due.